### PR TITLE
PV2-553 Add commitments to e2e tests

### PR DESCRIPTION
**Pull request by carlbm**

### CODE REVIEW

File: src/SFA.DAS.Payments.FundingSource.ServiceFabric/PublishProfiles/Cloud.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
                                     AzureActiveDirectory="true"
                                     ServerCertThumbprint="0123456789012345678901234567890123456789" />
   -->
-  <ClusterConnectionParameters ConnectionEndpoint="" />
-  <ApplicationParameterFile Path="..\ApplicationParameters\Cloud.xml" />
+  <ClusterConnectionParameters />
+  <ApplicationParameterFile Path="..\ApplicationParameters\Local.1Node.xml" />
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.FundingSource.ServiceFabric/PublishProfiles/Cloud.xml]: I think this change should be reverted back to Cloud.xml as per other service fabric services

File: src/SFA.DAS.Payments.ProviderPayments.ProviderPaymentsService/PackageRoot/ServiceManifest.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <ServiceTypes>
     <StatefulServiceType ServiceTypeName="ProviderPaymentsServiceActorServiceType">
       <Extensions>
-        <Extension Name="__GeneratedServiceType__" GeneratedId="fcc6ff14-9fe2-4cd1-8860-c56f0070416b|Volatile">
+        <Extension Name="__GeneratedServiceType__" GeneratedId="262c2fc4-cb5a-422c-8966-97fac891d901|Volatile">
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.ProviderPayments.ProviderPaymentsService/PackageRoot/ServiceManifest.xml]: does this need to be a new Guid?

File: src/SFA.DAS.Payments.ProviderPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 3]
```diff
@@ -62,7 +62,7 @@
         <SingletonPartition />
       </StatelessService>
     </Service>
-    <Service Name="ProviderPaymentsServiceActorService" GeneratedIdRef="fcc6ff14-9fe2-4cd1-8860-c56f0070416b|Volatile">
+    <Service Name="ProviderPaymentsServiceActorService" GeneratedIdRef="262c2fc4-cb5a-422c-8966-97fac891d901|Volatile">
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.ProviderPayments.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml]: does this need to be a new Guid?

File: src/SFA.DAS.Payments.ProviderPayments.ProviderPaymentsService/PackageRoot/ServiceManifest.xml (hunk 2 above)
Comment (carlbm) [reply src/SFA.DAS.Payments.ProviderPayments.ProviderPaymentsService/PackageRoot/ServiceManifest.xml]: Probably not, but I have no idea why it keeps getting regenerated when I change it back!

File: src/SFA.DAS.Payments.FundingSource.ServiceFabric/PublishProfiles/Cloud.xml (hunk 1 above)
Comment (carlbm) [reply src/SFA.DAS.Payments.FundingSource.ServiceFabric/PublishProfiles/Cloud.xml]: Changed back

**Pull request merged**